reference/tools.rst

```diff
@@ -1879,6 +1879,29 @@ This is a context manager that allows to append to the environment all the varia
         with tools.compilervars(self.settings):
             do_something()
 
+.. tools_intel_installation_path:
+
+tools.intel_installation_path()
```

Comment:
Better mark it as experimental, new tools and features deserve at least 1-2 releases, just in case some important change is needed.

Reply:
Otherwise, it is great. Feature has been merged in 1.30. Thanks very much!

Reply:
Done.